modules/shortcodes/vimeo.php

```diff
@@ -78,7 +78,22 @@ function vimeo_shortcode( $atts ) {
 		$height = round( ( $width / 640 ) * 360 );
 	}
 	
+	/**
+	 * Filter the Vimeo player width.
+	 *
+	 * @since 3.4-beta1
```

Comment:
I thought about this for a bit, and I don't know if there's a good solution.

The problem with using only stable versions is that those tags/releases may not even exist yet. Using this PR as an example, I wouldn't think we should document something as `@since 3.4.0` if there is [no such tag](https://github.com/Automattic/jetpack/tags) at the time we submit the PR.

OTOH, using `@since 3.4-beta1`, while technically correct, seems non-ideal as well. :confused: